pitch.md

```diff
@@ -1,60 +1,142 @@
+make them per hour. So, there is a lot of variability in the tasks and timelines, but they all follow a similar
+outline of big categories, smaller tasks, and a time scale.
 
 
 ## Why you?
-_What excites you about this idea? How did you come up with it?_
+_What excites you about this idea? How did you come up with it?_  
+I'm very excited about this project because of the usefulness of it. I know a lot of people who struggle with
+this same problem and I think that it would be very rewarding to have people use it (after, of course, the
+initial prototype struggles). 
+
+Initally, I thought of this idea because of my experiences dealing with multiple people having a bunch of
+tasks and having short, stressful periods followed by long, relaxed periods. As I have thought more 
+about this idea, though, I don't think this would be my main focus. I think that
+multiple people would add too much variability to the language and make it less focused. Right now, I
+think the language should focus on one person being more productive with their free time. By being able
+to map out when tasks need to be done for one person seems to be effective enough and overall more
+helpful because most people make to-do lists only for themselves.
 
 
 ## Interface (syntax)
 _What kinds of things can the user say in your language. How, generally, do they
 say it? **Be as specific as possible.** You don't need to design syntax; but you
 should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
-important to users and what kinds of values those properties might have._ 
+important to users and what kinds of values those properties might have._   
+The user should be able to name categories and give tasks with names and due dates in each of those
+categories. The user should also be able to specify a timeline that is consistent with the due
+dates of the tasks. For example, if a user says that the task is due Monday night, they should have
+a Monday night time slot so that the language can figure out where "Monday night" is in relation
+to the rest of the timeline. This way the user can have a to-do list that is the most helpful
+to them and allows for customizability.
+
+Additionally this is a rather simple syntax: users will specify categories, tasks (with a time),
+and a timeline so the users hopefully do not have to learn much to use this language.
 
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
-might they be communicated to the user?_
+might they be communicated to the user?_  
+When a program runs, it should output a list of things to do for each time period given
```

Comment:
Output as a plain text list?  This is what I would expect but it seems like there'd also be some value to a visual list.